opencog/nlp/types/atom_types.script

```diff
@@ -118,6 +118,7 @@ LG_CONN_DIR_NODE <- NODE    // e.g. "+"
 // Consists of the connector node, the direction, and possibly the
 // multi-connector node.
 LG_CONNECTOR <- LINK   // e.g. "MX+"
+MST_Connector <- LINK "MSTConnector"
 
```

Comment:
Didn't saw it, but lower case naming `MST_Connector` is breaking build.  Need to change to `MST_CONNECTOR`